assets/src/design-system/components/toggle/index.js

```diff
@@ -26,23 +26,30 @@ import { CheckmarkSmall } from '../../icons';
 import { FOCUS_VISIBLE_SELECTOR } from '../../theme/global';
 
 const BORDER_WIDTH = 1;
-const TOGGLE_HEIGHT = 20;
+const TOGGLE_TRACK_HEIGHT = 20;
 const TOGGLE_WIDTH = 44;
 const CIRCLE_DIAMETER = 28;
-const CIRCLE_INITIAL_POSITION = (TOGGLE_HEIGHT - CIRCLE_DIAMETER) / 2;
+const WRAPPER_PADDING = (CIRCLE_DIAMETER - TOGGLE_TRACK_HEIGHT) / 2;
+const CIRCLE_INITIAL_POSITION =
+  (TOGGLE_TRACK_HEIGHT - CIRCLE_DIAMETER) / 2 - BORDER_WIDTH;
 const CIRCLE_FINAL_POSITION =
   TOGGLE_WIDTH - CIRCLE_INITIAL_POSITION - CIRCLE_DIAMETER;
 
 const ICON_CONTAINER_WIDTH = 32;
 const ICON_TOP_POSITION = -6;
 const ICON_LEFT_POSITION = 19;
 
+const Wrapper = styled.div`
+  /* Account for circle leaking out of ToggleContainer */
+  padding: ${WRAPPER_PADDING}px;
+`;
```

Comment:
Adds a wrapper around the toggle so that the big circle is also accounted for in the space (since it is absolutely positioned). Decided to use padding, but we don't have to do it with padding if we want to do something different.

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/115474159-16b83900-a1fa-11eb-973b-2b5f80ae2f83.png)|![image](https://user-images.githubusercontent.com/22185279/115474194-28014580-a1fa-11eb-82bf-94c393ee03d1.png)|

Reply:
If you want, create a ticket to update this for the content box to contain the whole toggle and we'll through it in our `fixit` sprint after this next release.

Reply:
tracked here #7280